Symfony/CS/Fixer/PSR2/VisibilityFixer.php

```diff
@@ -123,7 +123,7 @@ private function grabAttribsBeforeMethodToken(Tokens $tokens, $index)
      * @param Tokens $tokens Tokens collection
      * @param int    $index  token index
      *
-     * @return array array of grabbed attributes
+     * @return array<string, Token|null> array of grabbed attributes
```

Comment:
I think we should remove the var_to_type fixer in 2.x then since `@type` isn't a thing anymore.